class AsyncException(Exception):
    """
    lambda-django-tasker exception
    """
